src/App.js

```diff
@@ -58,19 +70,18 @@ class App extends Component {
           <h2 className="page-name">Quizzelydoo</h2>
         </div>
         {isQuizIntro ? (
-          <div className="Introduction">
-            <Adbox />
+          <Transition in={this.state.in} timeout={400} className="exiting">
             <Introquiz
+              className="Quiz-Introduction"
               quiztitle={quizInfo.quizheadline.quiztitle}
               intropic={quizInfo.quizheadline.intropic}
               quizsummary={quizInfo.quizheadline.quizsummary}
               onClick={this.handleQuizStart}/>
-            <Adboxtwo />
-          </div>
+          </Transition>
         ) : (
-          <div className="Quiz-Display">
+          <Transition in={this.onEnter} timeout={400} className="entering">
```

Comment:
First off. The compiler gets confused when multiple children are used with a <Transition> so wrap everything in here with a <div> or pull the counter and results out of the isQuizIntro switch so there is only one element child of <Transition> (They can live outside of it).

As for your className: name it something about its content not how it's entering. Whatever className you put in here will be the base name for the four animation classes. If you put in "questions" for className you'd get .questions-entering .questions-entered .questions-exiting and .questions-exited as your classes.